File: 2019-01-02/1929/1929.c
#include <stdio.h>
#include <math.h>

void estoras(int num);

int main(){
	int M, N;
	scanf("%d %d", &M, &N);
	for(int i = M; i <= N; i ++){
		if(i != 1)
			estoras(i);
	}
}

void estoras(int num){
	for(int i = 2; i <= sqrt(num); i++){
		if(num % i == 0)
			return ;
	}
	printf("%d\n", num);
}

File: 2019-01-02/1929/1929.java
import java.io.*;

class Main{
	public static void main(String[] args) throws Exception{
		BufferedReader bfr = new BufferedReader(new InputStreamReader(System.in));
		String MN[] = bfr.readLine().split(" ");

		int M = Integer.parseInt(MN[0]);
		int N = Integer.parseInt(MN[1]);
		for(int i = M; i <= N; i++){
			if(i != 1){
				estoras(i);
			}
		}
	}

	private static int estoras(int num){
		for(int i = 2; i <= Math.sqrt(num); i++){
			if(num % i == 0){
				return 0;
			}
		}
		System.out.println(num);
		return 1;
	}
}